### PR TITLE
Create markdwon tables for query parameters in each operation

### PR DESCRIPTION
**Pull request by mapedraza**

### CODE REVIEW

File: doc/manuals/orion-api.md [hunk 1]
```diff
@@ -1184,26 +1158,23 @@ Response:
...
+|------------|----------|--------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------|
+| `entityId` |          | string | Id of the entity to be retrieved                                                                                                                                                                                                                                                                                                                                                        | `myRoom_112` |
```
Comment (fgalan) [inline doc/manuals/orion-api.md:1165]:
```suggestion
```

This table should include only query parameters. The id should be included in

```
#### Retrieve Entity [GET /v2/entities] -> #### Retrieve Entity [GET /v2/entities/{entityId}]
```

This is probably a typo in a previous PR. ToC should be also fixed

File: doc/manuals/orion-api.md [hunk 2]
```diff
@@ -1147,13 +1121,18 @@ Response:
...
         }
 
-+ Parameters
-    + options (optional, string) - Options dictionary
-      + Members
-          + keyValues - when used, the request payload uses the `keyValues` simplified entity
-            representation. See "Simplified Entity Representation" section for details.
-          + upsert - when used, entity is updated if already exits. If upsert is not used and
-            the entity already exist a `422 Unprocessable Entity` error is returned.
+**Parameters**
```
Comment (fgalan) [inline doc/manuals/orion-api.md:1124]:
```suggestion
**Query Parameters**
```

File: doc/manuals/orion-api.md [hunk 3]
```diff
@@ -1184,26 +1163,23 @@ Response:
...
-    + options (optional, string) - Options dictionary
-      + Members
-          + keyValues - when used, the response payload uses the `keyValues` simplified entity
-            representation. See "Simplified Entity Representation" section for details.
-          + values - when used, the response payload uses the `values` simplified entity
-            representation. See "Simplified Entity Representation" section for details.
-          + unique - when used, the response payload uses the `values` simplified entity
-            representation. Recurring values are left out.
-            See "Simplified Entity Representation" section for details.
+**Parameters**
```
Comment (fgalan) [inline doc/manuals/orion-api.md:1166]:
```suggestion
**Query Parameters**
```

File: doc/manuals/orion-api.md [hunk 4]
```diff
@@ -1004,72 +1004,46 @@ and/or those which match a query or geographical query (see [Simple Query Langua
...
-            HTTP header named `Fiware-Total-Count`.
-          + keyValues - when used, the response payload uses the `keyValues` simplified entity
-            representation. See "Simplified Entity Representation" section for details.
-          + values - when used, the response payload uses the `values` simplified entity
-            representation. See "Simplified Entity Representation" section for details.
-          + unique - when used, the response payload uses the `values` simplified entity
-            representation.
-            Recurring values are left out.
-            See "Simplified Entity Representation" section for details.
+**Parameters**
```
Comment (fgalan) [inline doc/manuals/orion-api.md:1013]:
```suggestion
**Query Parameters**
```

File: doc/manuals/orion-api.md [hunk 5]
```diff
@@ -1693,6 +1687,18 @@ Response code:
...
+| `notification` |          | object | An object that describes the notification to send when the subscription is triggered.         |
+| `expires`      | ✓        | string | Subscription expiration date in ISO8601 format. Permanent subscriptions must omit this field. |
+| `status`       |          | string | Either `active` (for active subscriptions) or `inactive` (for inactive subscriptions). If this field is not provided at subscription creation time, new subscriptions are created with the `active` status, which can be changed by clients afterwards. For expired subscriptions, this attribute is set to `expired` (no matter if the client updates it to `active`/`inactive`). Also, for subscriptions experiencing problems with notifications, the status is set to `failed`. As soon as the notifications start working again, the status is changed back to `active`.                                                                                              |
+| `throttling`   | ✓        | number | Minimal period of time in seconds which must elapse between two consecutive notifications.    |
+
+
+
```
Comment (mapedraza) [inline doc/manuals/orion-api.md:1701]: Revert this for a subsequent PR
```suggestion
```

**Comment by mapedraza** [comment]
Parameters in the documentation should be separated into 2:
- URL parameters, for instance, the entity ID
- Query parameters, like `options=keyValues`
Note that this PR removes the URL parameters from the query parameters

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**